src/components/ToolTip/ToolTip.scss

```diff
@@ -0,0 +1,31 @@
+.arrow-Top {
+	border-top: solid 10px #1f6b46;
+	border-left: solid 10px transparent;
```

Comment:
@frannkenstein remove the file

Reply:
why?

Reply:
@frannkenstein Tailwind doesn't require any scss file.